signposting.md

```diff
@@ -0,0 +1,123 @@
+        }
+      ],
+      "author": [
+        {
+          "href": "http://orcid.org/0000-0002-3748-8359"
+        },
+        {
+          "href": "https://isni.org/isni/0000002251201436"
+        }
+      ],
+      "cite-as": [
+        {
+          "href": "https://doi.org/10.1007/978-3-642-35233-1_18"
+        }
+      ],
+      "anchor": "https://{dspace.url}/handle/123456789/7"
+    },
+    {
+      "anchor": "https://{dspace.url}/bitstream/123456789/7/1/test.txt",
+      "collection": [
+        {
+          "href": "https://{dspace.url}/handle/123456789/7",
+          "type": "text/html"
+        }
+      ]
+    },
+    {
+      "anchor": "https://{dspace.url}/bitstream/123456789/7/1/test.pdf",
+      "collection": [
+        {
+          "href": "https://{dspace.url}/handle/123456789/7",
+          "type": "text/html"
+        }
+      ]
+    }
+  ]
+}
+```
+Return codes:
+* 200 OK - if the operation succeed
+* 403 Forbidden - if you are not logged in with sufficient permissions for reading Item information.
+* 404 Not found - if the item doesn't exist
+
+### in application/linkset serialization
+```/signposting/linksets/<:uuid>```
+
+This endpoint provides typed links of item having uuid specified in input, in application/linkset serialization
+
+```
+<https://{dspace.url}/bitstream/123456789/7/1/test.txt> ; rel="item" ; type="text/plain" ; anchor="https://{dspace.url}/handle/123456789/7" ,
+<https://{dspace.url}/bitstream/123456789/7/1/test.pdf> ; rel="item" ; type="application/pdf" ; anchor="https://{dspace.url}/handle/123456789/7" , 
+<http://orcid.org/0000-0002-3748-8359> ; rel="author" ; anchor="https://{dspace.url}/handle/123456789/7" ,
+<https://isni.org/isni/0000002251201436> ; rel="author" ; anchor="https://{dspace.url}/handle/123456789/7" , 
+<https://doi.org/10.1007/978-3-642-35233-1_18> ; rel="cite-as" ; anchor="https://{dspace.url}/handle/123456789/7" ,
+<https://{dspace.url}/handle/123456789/7> ; rel="collection" ; type="text/html" anchor="https://{dspace.url}/bitstream/123456789/7/1/test.txt" ,
+<https://{dspace.url}/handle/123456789/7> ; rel="collection" ; type="text/html" anchor="https://{dspace.url}/bitstream/123456789/7/1/test.pdf" ,
+```
+Return codes:
+* 200 OK - if the operation succeed
+* 403 Forbidden - if you are not logged in with sufficient permissions for reading Item information.
+* 404 Not found - if the item doesn't exist
+
+### linkset data for response header or meta tag
+```/signposting/links/<:uuid>```
+
+This endpoint provides typed links, of item or bitstream having uuid specified in input, to be included item or bitstream page response header or meta tag
+
+```json
+[
+  {
+    "href": "https://{dspace-url}/linksets/{uuid}/lset",
+    "rel": "linkset",
+    "type": "application/linkset"
+  },
+  {
+    "href": "https://{dspace-url}/linksets/{uuid}/json",
+    "rel": "linkset",
+    "type": "application/linkset+json"
+  },
+  {
+    "href": "https://{dspace-url}/handle/123456789/1",
```

Comment:
This one is a frontend link, so it should be `{dspace.ui.url}`.  It's important to remember that the frontend & backend *may not have the same URL*.  So, there is nothing that is "dspace.url" anymore.

Reply:
Has been replaced with `{dspace.ui.url}`